BUILD.md

```diff
@@ -172,6 +172,7 @@ on/off options currently supported by this repository:
 | BUILD_WSI_XLIB_SUPPORT | Linux | `ON` | Build the loader with the Xlib entry points enabled. Without this, the X11 headers should not be needed, but the extension `VK_KHR_xlib_surface` won't be available. |
 | BUILD_WSI_WAYLAND_SUPPORT | Linux | `ON` | Build the loader with the Wayland entry points enabled. Without this, the Wayland headers should not be needed, but the extension `VK_KHR_wayland_surface` won't be available. |
 | BUILD_WSI_DIRECTFB_SUPPORT | Linux | `OFF` | Build the loader with the DirectFB entry points enabled. Without this, the DirectFB headers should not be needed, but the extension `VK_EXT_directfb_surface` won't be available. |
+| BUILD_WSI_SCREEN_SUPPORT | QNX | `OFF` | Build the loader with the QNX Screen entry points enabled. Without this the extension `VK_QNX_screen_surface` won't be available. |
```

Comment:
`BUILD_WSI_SCREEN_SUPPORT` For anyone just looking at the list of build options this name is very non-descriptive. All the other WSI build options describe which api it is used for. The extension seems to use `VK_USE_PLATFORM_SCREEN_QNX` as the macro define, `BUILD_WSI_SCREEN_QNX_SUPPORT` would be similar.

Reply:
I agree. Will wait till you review all of it and will submit fixes.

Reply:
Ok, I updated this build option as you suggested. Thanks!